bin/create-content.py

```diff
@@ -63,6 +63,9 @@ def main(filename):
     if twitter_username:
         content += "twitter_username: %s\n" % twitter_username
 
+    hashtags = data["issue"]["hashtags"]
```

Comment:
Interesting I didn't know that Github injects markdown frontmatter fields into the event payload :+1: